numberformat/proposed.emu

```diff
@@ -180,8 +180,9 @@
         1. If _x_ is ~negative-zero~, then
           1. Let _isNegative_ be *true*.
           1. Let _x_ be the mathematical value 0.
-        1. Assert: _x_ is a mathematical value.
-        1. If _x_ &lt; 0, let _isNegative_ be *true*; else let _isNegative_ be *false*.
+        1. Else,
+          1. Assert: _x_ is a mathematical value.
+          1. If _x_ &lt; 0, let _isNegative_ be *true*; else let _isNegative_ be *false*.
```

Comment:
Given that `_x_` is a MV now, should this be `If _x_ &lt; 0` or `If ℝ(_x_) &lt; 0` ?

@gibson042